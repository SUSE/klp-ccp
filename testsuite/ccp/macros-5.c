#define A
#define B(a) # a
#define C(a) B(a)

void pu_f(void)
{
	C(A);
}
