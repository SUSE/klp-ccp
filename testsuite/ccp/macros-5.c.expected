#define A

#define B(a) # a

#define C(a) B(a)

void klpp_pu_f(void)
{
	C(A);
}
